ThroughPut/TP_BasePlate_Assembly.gscript

```diff
@@ -120,7 +128,7 @@ END
     #  $pos - center of part
     #  $phi - rotation of part in x-y plane
 
-    SETLIGHT 40
+    SETLIGHT &default.light_high_int
```

Comment:
I suggest for these to load these values at the top of the script with a `FLEXREAD` with a suitable default. See [here](https://github.com/CUASAS/pixel-gantry-control/blob/master/Gantry/gScript%20Application/gScript%20Documentation.md#flexread)

Reply:
I will change this and push my update.